src/library_gl.js

```diff
@@ -1003,7 +1019,7 @@ var LibraryGL = {
         // WebGL doesn't have GL_NUM_COMPRESSED_TEXTURE_FORMATS (it's obsolete since GL_COMPRESSED_TEXTURE_FORMATS returns a JS array that can be queried for length),
         // so implement it ourselves to allow C++ GLES2 code get the length.
         var formats = GLctx.getParameter(0x86A3 /*GL_COMPRESSED_TEXTURE_FORMATS*/);
-        ret = formats.length;
+        ret = formats?formats.length:0;
```

Comment:
And old experimental-webgl WebView browser was not supporting any compressed formats, and returns null here.